source/guide/3.2/guide-composer.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: bt_wiki
+title: Blueprint Composer User Guide
+category: Guides
+publish: true
+abstract:
+pageord: 800
+
+
+---
+{%summary%} {{page.abstract}}{%endsummary%}
+
+# Overview
+
+The Blueprint Composer is an editor for composing the blueprint.yaml dynamically using a modern Drag and Drop Interface.
+This guide will quickly walk you through the use of the Composer.
+
+{%tip title=Tip%}
+The [Terminology Reference Page]({{page.terminology_link}}) will help you understand some of the terms stated in this guide.
```

Comment:
the link doesn't work, you need something like this defined:
terminology_link: reference-terminology.html
(same applies for other links that start with "page.")